release/release_after_kokoro.sh

```diff
@@ -117,36 +123,62 @@ fi
 ###############################################################################
 echo
 echo "#"
-echo "# Copy 'gs://gcloud-for-eclipse-testing/category.xml'"
+echo "# Copy 'gs://gcloud-for-eclipse-testing/metadata.{product,p2.inf}'"
 echo "# into '$WORK_DIR'."
 echo "#"
 ask_proceed
 
 set -x
-gsutil cp gs://gcloud-for-eclipse-testing/category.xml $WORK_DIR
+# note that the metadata.p2.inf is renamed p2.inf as required for the p2 ProductPublisher
+gsutil cp gs://gcloud-for-eclipse-testing/metadata.product $WORK_DIR
+gsutil cp gs://gcloud-for-eclipse-testing/metadata.p2.inf $WORK_DIR/p2.inf
 set +x
 
-if [ ! -e "$WORK_DIR/category.xml" ]; then
-    echo "The file was not copied. Halting."
+if [ ! -e "$WORK_DIR/metadata.product" -o ! -e "$WORK_DIR/p2.inf" ]; then
+    echo "The files were not copied. Halting."
     exit 1
 fi
 
 ###############################################################################
 echo
 echo "#"
-echo "# Run org.eclipse.equinox.p2.publisher.CategoryPublisher."
+echo "# Run org.eclipse.equinox.p2.publisher.ProductPublisher to add any"
+echo "# additional p2 metadata for the CT4E repository:
+echo "#   - copyright and license have been associated with our public feature
+echo "# Verify using xmlstarlet."
 echo "#"
 ask_proceed
 
 set -x
 $ECLIPSE_HOME/eclipse \
     -nosplash -console -consolelog \
-    -application org.eclipse.equinox.p2.publisher.CategoryPublisher \
+    -application org.eclipse.equinox.p2.publisher.ProductPublisher \
     -metadataRepository file:$LOCAL_REPO \
-    -categoryDefinition file:$WORK_DIR/category.xml \
-    -categoryQualifier
+    -productFile $WORK_DIR/metadata.product \
+    -flavor tooling \
+    -append
 set +x
 
+# Validate license and copyright
+repoName="Google Cloud Tools for Eclipse"
+categoryId=com.google.cloud.tools.eclipse.category
+featureId=com.google.cloud.tools.eclipse.suite.e45.feature.feature.group
+copyrightText="Copyright 2016, 2017 Google Inc."
+licenseUri=https://www.apache.org/licenses/LICENSE-2.0
+licensesText="Cloud Tools for Eclipse is made available under the Apache\
+ License, Version 2.0. Please visit the following URL for details:\
+ https://www.apache.org/licenses/LICENSE-2.0"
+
+valid=$(unzip -p gcp-repo/target/repository/content.jar \
+  | xml sel -t -m "/repository[@name='$repoName']/units/unit[@id='$categoryId']" \
```

Comment:
Turns out `xml` is not an alias or a symlink of `xmlstarlet` on our machines. We need to call `xmlstarlet` directly.